gradoop-flink/src/test/java/org/gradoop/io/impl/tlf/TLFIOTest.java

```diff
@@ -0,0 +1,323 @@
+/*
+ * This file is part of Gradoop.
+ *
+ * Gradoop is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Gradoop is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Gradoop. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.gradoop.io.impl.tlf;
+
+import org.gradoop.io.api.DataSink;
+import org.gradoop.io.api.DataSource;
+import org.gradoop.model.GradoopFlinkTestBase;
+import org.gradoop.model.impl.GraphTransactions;
+import org.gradoop.model.impl.pojo.EdgePojo;
+import org.gradoop.model.impl.pojo.GraphHeadPojo;
+import org.gradoop.model.impl.pojo.VertexPojo;
+import org.gradoop.model.impl.tuples.GraphTransaction;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.Comparator;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+
+public class TLFIOTest extends GradoopFlinkTestBase {
+
+  @Rule
+  public TemporaryFolder currentFolder;
+  /**
+   * Test method for
+   *
+   * {@link TLFDataSource#getGraphTransactions()}
+   * @throws Exception
+   */
+  @Test
+  public void testFromTLFFile() throws Exception {
```

Comment:
if possible combine testFromTLFFile and testFromTLFFileWithEdgeCheck in one Test